core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -138,7 +138,7 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
               if (shouldShowCompletedJobs) {
                 <li>
                   <a href="#completed"><strong>Completed Jobs:</strong></a>
-                  {completedJobs.size}
+                  {listener.numCompletedJobs}
```

Comment:
yeah, I was just trying to make this consistent with the behavior in AllStagesPage, but I can update them both to have the behavior you suggest